dev-util/bats/bats-1.4.1.ebuild

```diff
@@ -13,15 +13,24 @@ SRC_URI="https://github.com/${MY_PN}/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+IUSE="test"
```

Comment:
Declaring the 'test' use flag is not necessary if there are no test dependencies. See also https://devmanual.gentoo.org/ebuild-writing/functions/src_test/index.html#supporting-test-suites-in-packages

Reply:
duplicate